walkthroughs/week-1-web-development/portfolio-walkthrough.md

```diff
@@ -176,9 +176,10 @@ file to contain a `<p>` element that introduces you. Then run a dev server and
 confirm that you can see your change.
 
 ## Uploading Images
```

Comment:
Can you please add a newline after this heading, so it matches the rest of the file? Thanks!

Reply:
Added a new line, and instructions for how to actually create the directory.